### DISCUSSION TITLE
Cronjob is executed, but there are no results visible

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#5599] [Firefly III] [answered]
Hi there,

I'm running firefly as a docker container on portainer. I putty into portainer, do "crontab -e" and filled in this "0 3 * * * docker exec --user www-data 94cdb269d49e418c1ce39758febc7c6e0e37deddd13b693e17cb92877ff073c6 /usr/local/bin/php /var/www/html/artisan firefly-iii:cron" . 

When i execute "grep CRON /var/log/syslog" i can clearly see that it has run at the requested time. 
But it actualy didn't cause firefly still gives me the message that i has been x amount of time since last run.
I've exectued the docker exec command manually to see if there is an error in this, but it just runs fine when i do it manually.

Jan 23 03:00:01 portainer CRON[158353]: (dieter) CMD (docker exec --user www-data 94cdb269d49e418c1ce39758febc7c6e0e37deddd13b693e17cb92877ff073c6 /usr/local/bin/php /var/www/html/artisan firefly-iii:cron)
Jan 23 03:00:01 portainer CRON[158354]: (root) CMD (docker exec --user www-data 94cdb269d49e418c1ce39758febc7c6e0e37deddd13b693e17cb92877ff073c6 /usr/local/bin/php /var/www/html/artisan firefly-iii:cron)

Ran twice, one for local user and one for root. Not sure if this brings issues.

**Comment by JC5** [comment]
The command is executed, but what are you getting back from Firefly III? Is there anything in the logs?

**Reply by ghost** [reply]
Where are the Firefly logs located?

**Reply by JC5** [reply]
In case of Docker, stdout.

**Reply by ghost** [reply]
Nothing is happening around my cronjobs timestamp

**Reply by JC5** [reply]
And what is the output from your cron command? Except the line you referred to already.

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/43341550/151234851-d76ccc76-5b1f-4944-a862-7342cf4bf3c4.png)

**Reply by JC5** [reply]
Not what I meant, unfortunately. This is a log of the call itself. But any output returned from the call, that's what I would like to see.

**Reply by ghost** [reply]
This is the only related stuff i found in the Syslog. I was looking for output of from the call also but haven't found it either. Only when i run the docker exec command manually do i get an output

**Reply by ghost** [reply]
I found some suggestions online that said i should pipe the error log to a log file so i did an i got.
`root@portainer:/# cat /var/log/fireflycron.log

/bin/sh: 1: docker: not found
`

**Reply by ghost** [reply]
Think i solved the issue.

-> Made a Cronjob that calls the url with the token. The banner message saying it has been over 36hour is gone so im 99% certain its solved :)

**Comment by JC5** [comment]
I edited your title to be more useful.

**Comment by ghost** [comment, marked as answer]
Think i solved the issue.

-> Made a Cronjob that calls the url with the token. The banner message saying it has been over 36hour is gone so im 99% certain its solved :)
